IPC/shared_memory_reader.c

```diff
@@ -0,0 +1,22 @@
+
+#include <iostream>
+#include <sys/ipc.h>
+#include <sys/shm.h>
+#include <stdio.h>
+using namespace std;
```

Comment:
What is this doing here? This is C, not C++, and `using namespace std;` is considered bad practice too.

Reply:
okay i understand 
so this cant be added because this repo focuses on having cross-platform compatibility